crates/zkevm-succinct/program/src/main.rs

```diff
@@ -14,8 +16,8 @@ pub fn main() {
 
     println!("cycle-tracker-report-start: read_input");
     let input = sp1_zkvm::io::read::<ClientInput>();
-    let network = sp1_zkvm::io::read::<ForkSpec>();
-    let chain_spec = Arc::new(network.into());
+    let genesis = sp1_zkvm::io::read::<Genesis>();
+    let chain_spec = Arc::new(genesis.into());
```

Comment:
We no longer rely on forkspec, so that our fork does not need to pull it out of ef-tests